packages/core/entities-config-editor/src/types/index.ts

```diff
@@ -0,0 +1,5 @@
+// Export all types and interfaces from this index.ts
+// The actual types and interfaces should be contained in separate files within this folder.
+
+// Example:
+// export * from './component-types'
```

Comment:
@sumimakito this file needs an export -- would you mind updating the CLI template as well in this PR? 
```suggestion
// export * from './component-types'

export {}
```

Reply:
Sure, no problem. I have fixed the export and updated the template.